app/src/components/layout/Header.vue

```diff
@@ -22,7 +22,7 @@
       </v-btn>
 
       <v-flex xs10 sm8 md6 style="background: white; border-radius: 20px; margin-right:2%">
-        <search-bar @input="addSearchValueAndUpdate" :currentSearchValue="currentSearchValue"/>
```

Comment:
Without this input event adding tags via 'Enter' is not possible

Reply:
It shouldnt be executed on input though. I think it currently doesnt work with enter too. Mapping on @input also disrupts the button functionality since this would cause the query to go off and then the button query going off resulting in two queries often times (one being empty often). Played around for a bit with @keyup / keypressed etc. but had not had much success. So i decided to tackle 'enter' search later

Reply:
Neither selection from dropdown nor input via enter is working
@see https://github.com/hochschule-darmstadt/einander-helfen/pull/68#issuecomment-647797225

Reply:
Well, gotta find the correct abstractions where all three methods work

Reply:
I actually think this should be its own issue considering
1. This requirement is never mentioned in any issue this detailed
2. This entails fixing problems like not searching on input for the searchbar, which is its own issue as a whole that was never addressed in any issue.